src/components/checkout/stripe-payment.js

```diff
@@ -90,7 +90,7 @@ class StripePayment extends Component {
 
         { !billingAsShipping &&
           <>
-            { addressBook.length && <this.AddressBook
+            { addressBook.length !== 0 && <this.AddressBook
```

Comment:
Is it worth making it `addressBook.length > 0`? Simply because if `addressBook.length` is undefined (because the addressBook doesn't come back as an array for whatever reason) then:
```
undefined !== 0 // => true
```
but
```
undefined > 0 // => false
```